integration_tests/sdk/delete_workflow_test.py

```diff
@@ -136,29 +139,48 @@ def test_delete_workflow_saved_objects_twice(client):
         flow_2_id = flow_list[1]
 
         tables = client.flow(flow_1_id).list_saved_objects()
-        assert "delete_table" in [item.object_name for item in tables[get_integration_name()]]
+        tables_1 = set([item.object_name for item in tables[get_integration_name()]])
+        assert "delete_table" in tables_1
 
         tables = client.flow(flow_2_id).list_saved_objects()
-        assert "delete_table" in [item.object_name for item in tables[get_integration_name()]]
+        tables_2 = set([item.object_name for item in tables[get_integration_name()]])
+        assert "delete_table" in tables_2
 
-        # No SDK function to do this so we query the endpoint directly to see delete_table is properly created at the integration.
-        tables_response = get_response(client, endpoint).json()
-        assert "delete_table" in set(tables_response["object_names"])
+        assert tables_1 == tables_2
+
+        # Check delete_table is properly created at the integration.
+        integration.sql("SELECT * FROM delete_table").get()
 
         client.delete_flow(flow_1_id, saved_objects_to_delete=tables, force=True)
 
+        flow_ids_to_delete.remove(flow_1_id)
+
         # Wait for deletion to occur
-        sleep(1)
+        timeout = 500
```

Comment:
nit: I think like 60 seconds is good enough. We also don't want to hang for too long and waste time :P